src/sortedDict.jl

```diff
@@ -241,6 +243,8 @@ immutable IncludeLast{K, D, Ord <: Ordering}
     last::Int
 end
 
+eltype{K,D,Ord<:Ordering}(e::IncludeLast{K,D,Ord}) = eltype(e.m)
+
```

Comment:
On the other hand, this one (and the one above) doesn't need any type parameters because they are not used.

The constraint on the `Ord` parameter can also be removed for the ones below.